### PR TITLE
Add modman support for Magento1

### PR DESCRIPTION
**Pull request by norgeindian**
With this approach it would also be possible to handle modman environments in Magento1.
Solves #253

### CODE REVIEW

**Comment by sprankhub** [comment]
Looks like an easy change. However, IMHO, we could also auto-detect modman-environments. If there is a `.modman` directory in the Warden root, we have a modman-environment. And in this case, `.modman/.basedir` already contains the necessary document root. Would IMHO be nicer to auto-detect this and not introduce another setting for this.

**Comment by norgeindian** [comment]
@sprankhub , thanks for the idea.
Adapted my pull request, so modman environments are now automatically detected and handled.
What do you think about this approach?

File: commands/env.cmd [hunk 1]
```diff
@@ -29,6 +29,12 @@ if [[ ${WARDEN_ENV_TYPE} != local ]]; then
     WARDEN_REDIS=${WARDEN_REDIS:-1}
 fi
 
+if [[ ${WARDEN_ENV_TYPE} == "magento1" && -f "${WARDEN_ENV_PATH}/.modman/.basedir" ]]; then
+  export WARDEN_MODMAN_ROOT='/'`cat "${WARDEN_ENV_PATH}/.modman/.basedir"`
```
Comment (sprankhub) [inline commands/env.cmd:33]: I am not a bash expert at all, but https://www.shellcheck.net/ recommends to [not use backticks](https://github.com/koalaman/shellcheck/wiki/SC2006). `'/'$(cat "${WARDEN_ENV_PATH}/.modman/.basedir")` should be a proper alternative.

File: commands/env.cmd [hunk 2]
```diff
@@ -29,6 +29,12 @@ if [[ ${WARDEN_ENV_TYPE} != local ]]; then
...
+else
+  export WARDEN_MODMAN_ROOT='/'
```
Comment (sprankhub) [inline commands/env.cmd:35]: I think the name is weird. Yes, we use it for a `modman`-specific fix, but it is used in all cases.

1. Maybe we can directly set it to `NGINX_PUBLIC`?
2. If not, maybe we use a generic name like `NGINX_PUB_DIR`?

File: commands/env.cmd (hunk 1 above)
Comment (norgeindian) [reply commands/env.cmd]: Thanks, adpated

File: commands/env.cmd (hunk 2 above)
Comment (norgeindian) [reply commands/env.cmd]: Directly is not possible, at least I did not find a way.
So I find this naming already better, you're right

**Comment by sprankhub** [comment]
Nice! Let's see what @davidalger says ;-)

**Review by davidalger — COMMENTED** [review]
Sounds like a great idea to me. Left a couple comments / nit-picks. Once those are addressed, be happy to merge it. Thanks!

**Comment by norgeindian** [comment]
@davidalger , thanks a lot for your comments and suggestions.
I included them, so feel free to merge.

**Pull request merged**